Xamarin.PropertyEditing.Mac/Controls/CombinablePropertyEditor.cs

```diff
@@ -1,16 +1,16 @@
 using System;
 using System.Collections;
-using System.Diagnostics;
+using System.Collections.Generic;
+using System.Linq;
 using System.ComponentModel;
 
 using Foundation;
 using AppKit;
 using CoreGraphics;
 
+using Cadenza.Collections;
```

Comment:
What is Cadenza??

Reply:
It's where `OrderedDictionary` comes from